src/writer.rs

```diff
@@ -75,10 +79,11 @@ impl fmt::Display for Error {
     }
 }
 
+#[cfg(feature = "std")]
```

Comment:
Maybe we can implement https://doc.rust-lang.org/beta/core/error/trait.Error.html for no_std? WDYT?

Reply:
I'd like to avoid requiring `nightly` for `no_std`. We could add another `nightly` feature, but I don't see a big benefit in this.

Reply:
Oooops, I didn't see that you need nightly for that. That's a good call!